documents/motions/10.2024.3 - Integrity Committee.md

```diff
@@ -0,0 +1,41 @@
+         1. The subjects of the investigations are incidents during WCA Competitions, incidents related to WCA Competitions, or incidents within the WCA Community. Incidents are alleged violations of the Spirit, Mission, Motions, Policies, Codes and/or Regulations of the WCA.
+         2. The investigations can be initiated by the WCA Integrity Committee itself or by the WCA Board.
+      2. Performing independent investigations regarding WCA Regional Organizations.
+         1. The subjects of the investigations are incidents during WCA Competitions, incidents related to WCA Competitions, incidents related to the function of the Regional Organizations or incidents within the WCA Community. Incidents are alleged violations of the Spirit, Mission, Motions, Policies, Codes and/or Regulations of the WCA.
+         2. The investigations can be initiated by the WCA Integrity Committee itself or by the WCA Board.
+      3. Delivering an independent report on the investigations.
+      4. Making decisions on disciplinary actions in relation to the investigated incidents. See [WCA Motion : Suspensions and other Sanctions](wcadoc{documents/motions/15.2022.1 - Suspensions and other Sanctions.pdf}).
+   2. Confidant
+      1. Being the confidential contact for WCA Community members, in case of important disciplinary and ethical matters that are related to WCA activities.
+   3. Information and administration
+      1. Informing the WCA Board on investigations, decisions, and disciplinary actions.
+      2. Publishing public information on the investigations, decisions, and disciplinary actions where necessary.
+      3. Managing and administrating investigations, reports, decisions, and disciplinary actions.
+      4. Keeping all information confidential that needs to stay confidential, based on internal and external guidelines
+   4. Document maintenance
+      1. Maintaining and publishing a Code of Ethics that all WCA Staff must abide by.
+      2. Maintaining and publishing a Code of Conduct that all members of the WCA Community must abide by.
+3. All WCA Members must fully cooperate with investigations by the WCA Integrity Committee.
+   1. All WCA Members should respond to communications sent on behalf of the WCA Integrity Committee within one week of each message.
+      1. A WCA Member must inform the WCA Integrity Committee if they decide to stop responding to the WCA Integrity Committee and explain as to the reasons why.
+      2. The WCA Integrity Committee may place temporary sanctions on WCA Members who cease cooperating with the WCA Integrity Committee.
+5. In case the WCA Integrity Committee is not constituted or for any reason non-functional, the WCA Appeals Committee will act as WCA Integrity Committee.
+   1. The WCA Appeals Committee shall be deemed non-functional for any investigation where it is unable to cary out the investigation process in an unbiased manner.
```

Comment:
Should this be referring to the Integrity Committee instead of the Appeals Committee?

From how I am reading it, this line is saying that a conflict makes the Integrity Committee non functional and there is an almost identical line to this in the Appeals document on line 27